src/validation/UserSchema.tsx

```diff
@@ -1,8 +1,13 @@
 import { object, string } from "yup";
 import { t } from "@lingui/macro";
 
+const EmailRegex =
```

Comment:
Any regex in code needs to have a suite of tests to prove it's correctness. Regex's are too often write-only code :(. Personally I can't read from this why we're listing those three domains in there.

Is there a particular reason why we couldn't use [yup's `.email()` validation](https://github.com/jquense/yup?tab=readme-ov-file#stringemailmessage-string--function-schema)?

Reply:
Wasn't aware of that option, will check it out